src/Microsoft.TemplateEngine.Cli/LocalizableStrings.cs

```diff
@@ -5,27 +5,27 @@ namespace Microsoft.TemplateEngine.Cli
 {
     internal class LocalizableStrings
     {
-        public const string ShowsAllTemplates = "Shows all templates";
+        public const string ShowsAllTemplates = "Shows all templates.";
 
         public const string DisplaysHelp = "Displays help for this command.";
 
-        public const string ParameterNamePrefixError = "Parameter names must start with -- or -";
+        public const string ParameterNamePrefixError = "Parameter names must start with -- or -.";
 
-        public const string ParameterReuseError = "Parameter name {0} cannot be used for multiple purposes";
+        public const string ParameterReuseError = "Parameter name {0} cannot be used for multiple purposes.";
 
-        public const string MultipleValuesSpecifiedForSingleValuedParameter = "Multiple values specified for single value parameter: {0}";
+        public const string MultipleValuesSpecifiedForSingleValuedParameter = "Multiple values specified for single value parameter: {0}.";
 
-        public const string ValueSpecifiedForValuelessParameter = "Value specified for valueless parameter: {0}";
+        public const string ValueSpecifiedForValuelessParameter = "Value specified for valueless parameter: {0}.";
 
-        public const string ParameterSpecifiedMultipleTimes = "Parameter [{0}] was specified multiple times, including with the flag [{1}]";
+        public const string ParameterSpecifiedMultipleTimes = "Parameter [{0}] was specified multiple times, including with the flag [{1}].";
 
-        public const string ParameterMissingValue = "Parameter [{0}] ({1}) must be given a value";
+        public const string ParameterMissingValue = "Parameter [{0}] ({1}) must be given a value.";
 
-        public const string TemplateMalformedDueToBadParameters = "Template is malformed. The following parameter names are invalid: {0}";
+        public const string TemplateMalformedDueToBadParameters = "Template is malformed. The following parameter names are invalid: {0}.";
 
         public const string OptionVariantAlreadyDefined = "Option variant {0} for canonical {1} was already defined for canonical {2}";
 
-        public const string ListsTemplates = "List templates containing the specified name.";
+        public const string ListsTemplates = "Lists templates containing the specified argument. If no argument is specified, lists all templates.";
```

Comment:
I think this makes it sound like the switch takes an argument rather than referring to the argument passed prior to any switch specification - it could just be me though